package.json

```diff
@@ -24,6 +24,8 @@
     "smartdc-auth": "2.1.6"
   },
   "devDependencies": {
+    "ldapjs": "^1.0.0",
```

Comment:
Perhaps just dep on the smaller `ldap-filter` <https://github.com/pfmooney/node-ldap-filter>?